doc/sphinx/proof-engine/ltac2.rst

```diff
@@ -923,8 +923,8 @@ Due to conflicts, a few syntactic rules have changed.
 
 - The dispatch tactical :n:`tac; [foo|bar]` is now written :n:`tac > [foo|bar]`.
 - Levels of a few operators have been revised. Some tacticals now parse as if
-  they were a normal function, i.e. one has to put parentheses around the
-  argument when it is complex, e.g an abstraction. List of affected tacticals:
+  they were a normal function; that is, one has to put parentheses around the
```

Comment:
were normal functions.  Parentheses are now required around complex arguments such as abstractions.

Reply:
yes, that is much better